apps/docs/content/components/buttons/SegmentedControl.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: SegmentedControl
+description: The SegmentedControl component presents a horizontal row of options or actions that are contextually or conceptually related. It allows users to select a single option at a time.
+category: "pickers"
+links:
+    source: https://github.com/gsoft-inc/wl-hopper/blob/main/packages/components/src/SegmentedControl/src/SegmentedControl.tsx
+    aria: https://www.w3.org/WAI/ARIA/apg/patterns/checkbox/
+---
+
+<Example src="SegmentedControl/docs/preview" isOpen />
+
+## Usage
+
+### Selected
+A segmented control supports single selection. At any given time, only one item can be active.
```

Comment:
The text and the examples don't seem to match to well. 

"A segmented control supports single selection. At any given time, only one item can be active." is a sentence for the control, not the individual example you're showcasing. We should say something along the line that a segment can be selected by default